packages/app/src/__tests__/utils.test.ts

```diff
@@ -73,3 +80,78 @@ describe('formatAttributeClause', () => {
     );
   });
 });
+
+describe('getMetricTableName', () => {
+  // Base source object with required properties
+  const createBaseSource = () => ({
+    from: {
+      tableName: 'default_table',
```

Comment:
nit: as I recall, the tableName is an empty string for the metrics source

Reply:
I meant to overwrite that oops. Fixing rq